app/directives/side-events/unscheduled.js

```diff
@@ -324,6 +326,25 @@ define(['app', 'lodash', 'text!./unscheduled.html', 'moment', 'text!../forms/edi
                         } //initMeeting
                         $scope.load=load;
 
+                    function getReqCode(res){
+                        let code = '';
+                        
+                        const { sideEvent } = res;
+                        const { requirements } = sideEvent;
+
+                        const isHybrid = requirements?.hybrid?.required;
+                        const isRemote = requirements?.interpretationType === 'online';
+                        const isOnsite = requirements?.interpretationType === 'onsite';
+                        const isInterpret = requirements?.interpretation && (!isRemote && !isOnsite);
```

Comment:
are you sure you want to use `?`, there is no rollup compilation in this project